javalib/src/main/scala-3/java/lang/ProcessBuilder.scala

```diff
@@ -0,0 +1,194 @@
+// Any changes to this files needs to be synced with it's Scala2 counter part.
```

Comment:
```suggestion
// Any changes to this file need to be synced with it's Scala2 counterpart.
```
Typo, here and in other scala-3 files